app/serializers/user_serializer.rb

```diff
@@ -1,11 +1,20 @@
 class UserSerializer
   include JSONAPI::Serializer
 
-  attributes :email, :google_id, :google_token, :google_image_url, :zip_code,
-             :summary, :goal, :availability_morning, :availability_afternoon,
-             :availability_evening, :first_name, :last_name, :full_name
+  attributes :email, :google_id, :google_image_url, :zip_code, :summary, :goal,
+             :availability_morning, :availability_afternoon,
+             :availability_evening, :full_name
 
   # TODO: Add relationships so they can be included in serialized hash
```

Comment:
The use of metadata on this will be an awesome solution.  Nice working ahead on this